src/pages/general/login/components/loginForm.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {LoginCredentials} from '../../../../model/loginCredentials';
+
+interface Props {
+    loginCredentials: LoginCredentials,
+    updateLoginInfo: (loginCredentials:LoginCredentials) => void;
+    performLogin: () => void;
+}
+
+export const LoginFormComponent = (props: Props) => {
+    return(
+        <div className="panel-body">
+          <form role="form">
+            <fieldset>
+              <div className="form-group">
+                <input className="form-control" 
+                       placeholder="E-mail" 
+                       name="email" 
+                       type="text"
+                       value={props.loginCredentials.login}
+                       onChange={(e : any) => props.updateLoginInfo({login: e.target.value, password: props.loginCredentials.password })}
```

Comment:
There is a lint rule to avoid lambdas in components attributes. `I solved it in my code doing this:
```javascript
function updateStudentFullName(e: any) {
  this.props.updateStudent('fullname', e.target.value);
}

export const EditStudentComponent = (props: IProps) => {
  return (
    <div>
      <input
        type="text"
        value={props.student.fullname}
        onChange={updateStudentFullName}
      />
      ...
    </div>
  );
};
```
BTW in the parent component I mannually create a new object and update the corersponding property:
```javascript
private updateStudent(property: string, value: any) {
    // Clone the object
    this.props.student = {
      id: this.props.student.id,
      fullname: this.props.student.fullname,
      email: this.props.student.email,
      phoneNumber: this.props.student.phoneNumber,
      isActive: this.props.student.isActive,
    };

    this.props.student[property] = value;
  }
```
But not sure if this is the way to go or we better use deep clone.